src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -1,3 +1,4 @@
+<html xmlns:th="http://www.thymeleaf.org">
```

Comment:
I added Thymeleaf library to reusablecomponents.html file to fix the error "Namespace 'th' is not bound".
Sources:
* Lecture
* https://www.javaer101.com/en/article/13765676.html